ui/components/NodeInputSubmit.tsx

```diff
@@ -9,15 +9,59 @@ export const NodeInputSubmit: FC<NodeInputProps> = ({
   setValue,
   disabled,
   dispatchSubmit,
-}) => (
-  <Button
-    onClick={async (e) => {
-      // On click, we set this value, and once set, dispatch the submission!
-      await setValue(attributes.value as string).then(() => dispatchSubmit(e));
-    }}
-    disabled={attributes.disabled || disabled}
-    className="login-button"
-  >
-    {getNodeLabel(node)}
-  </Button>
-);
+}) => {
+  const getProviderImage = (value: string) => {
+    if (value.toLowerCase().startsWith("auth0")) {
+      return "logos/Auth0.svg";
+    }
+    if (value.toLowerCase().startsWith("github")) {
+      return "logos/Github.svg";
+    }
+    if (value.toLowerCase().startsWith("google")) {
+      return "logos/Google.svg";
+    }
+    if (value.toLowerCase().startsWith("microsoft")) {
+      return "logos/Microsoft.svg";
+    }
+    if (value.toLowerCase().startsWith("okta")) {
+      return "logos/Okta.svg";
+    }
```

Comment:
There is no Okta provider type in Kratos, let's remove this and the Okta icon logo  (see https://www.ory.sh/docs/kratos/social-signin/overview)

Reply:
removed the entry for okta.